ghost/core/core/frontend/src/ghost-stats/ghost-stats.bundled.js

```
@@ -0,0 +1,545 @@
+    "email",
+    "address",
+    "phone",
+    "sex",
+    "gender",
+    "order",
+    "order_id",
+    "orderid",
+    "payment",
+    "credit_card"
+  ];
+  function maskSensitiveData(payload, attributesToMask = SENSITIVE_ATTRIBUTES) {
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)